valhalla/mjolnir/traffic_updater.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <future>
+#include <optional>
+#include <string>
+#include <thread>
```

Comment:
Looks like `#include <thread>` should be moved into src/mjolnir/traffic_updater.cc